cli/src/torii/routing.rs

```diff
@@ -361,28 +357,28 @@ mod subscription {
 #[cfg(feature = "telemetry")]
 async fn handle_version(sumeragi: Arc<Sumeragi>) -> Json {
     use iroha_version::Version;
-    /*
-        TODO REDO
-        #[allow(clippy::expect_used)]
-        reply::json(
-            &wsv.blocks()
-                .last()
-                .expect("At least genesis should always exist")
-                .value()
-                .version()
-                .to_string(),
-    )
-         */
-    warp::reply::json(b"Not implemented")
+
+    let wsv = sumeragi.wsv_clone();
```

Comment:
There's a small discussion to be had about how `wsv.clone()` actually lets you do the same. Interior mutability of `wsv` means that we might as well make `wsv` a public member, given that any getter can use this to clone and to modify.